### ISSUE TITLE
Plotting errors

### ISSUE THREAD

**Issue by abielr** [IRkernel/IRkernel#118]
```
library(ggplot2)
plot(1:10)
qplot(1:10)
```
If either of the charting commands are run alone in their own cell then they correctly yield a single chart. However, when run in one cell, it yields the first chart, the second chart, and then another copy of the first chart.

In addition, the `plot()` command shows a `Error in replayPlot(obj): invalid graphics state` warning, while the `qplot()` command yields a `Error in if (args[[1]]$name == "C_title" && !is.null(args[[2]])) {: missing value where TRUE/FALSE needed` warning.

**Comment by yhtgrace** [comment]
`Error in if (args[[1]]$name == "C_title" && !is.null(args[[2]])) {: missing value where TRUE/FALSE needed` is from `repr`'s `repr_text.recorded_plot` for getting the plot title. As a workaround, I've been checking if `(args[[1]]$name == "C_title" && !is.null(args[[2]]))` returns `NULL`.

**Comment by flying-sheep** [comment]
i tested a bit, and R really is a mess.

* `args[[1]]$name` can be `NULL`. then
* `NULL == "C_title"` is `logical(0)`. and therefore
* `logical(0) && TRUE` is `NA`
* `logical(0) && FALSE` is `FALSE`
* `if (NA)` yields this error.

so checking if it’s NULL isn’t the right thing, but we can do:

```r
if (isTRUE(args[[1]]$name == "C_title") && !is.null(args[[2]])) {
	...
}
```

or

```r
if (!is.null(args[[1]]$name) && args[[1]]$name == "C_title" && !is.null(args[[2]])) {
	...
}
```

i hope that‘s enough safeguarding.

**Comment by flying-sheep** [comment]
fixed in IRkernel/repr@83d435f3a6052cf805b0d08bc6ee92f4e19470fb

**Comment by flying-sheep** [comment]
the other part is already #113
